src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_redirect.html.twig

```diff
@@ -39,7 +39,7 @@
             <div class="block-exception">
                 <h1>Redirection Intercepted</h1>
 
-                {% set absolute_url = host in location ? location : host ~ location %}
+                {% set absolute_url = host in location or not (location starts with '/') ? location : host ~ location %}
```

Comment:
`//protocol-relative ` URLs are a thing :) Also, `.` is a hint of a relative URL.